### PR TITLE
files for code review of my own repo

### PR DESCRIPTION
**Pull request by Eckybrazzz**
In this lesson, I have to show that I master the code review on my own code with inline comments.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,892 @@
...
+					
+						<g class="style-scope yt-icon">
+							<path
+								d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
+								class="style-scope yt-icon"
+							></path>
+						</g>
+					</svg>
+				</button>
+				<button class="btn home-btn">
```
Comment (Eckybrazzz) [inline index.html:36]: I would suggest that you don't leave empty lines in your code.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,714 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.btn .microphone {
+  height: 2.5rem !important;
+  width: 2.6rem !important;
+  fill: var(--primaryColor);
+  margin-left: -3rem;
```
Comment (Eckybrazzz) [inline style.css:93]: I would suggest using the `!important` only as the last resource to fix your css

File: style.css [hunk 3]
```diff
@@ -0,0 +1,714 @@
...
+    margin: 0 1.5rem;
+  }
+}
+
+.auth .profile {
+  display: block;
+  width: 3.2rem !important;
+  height: 3.2rem !important;
+  margin-right: 4rem;
+}
```
Comment (Eckybrazzz) [inline style.css:158]: As before, I would suggest to use the `!important`  with care if you can't find another option to fix your css

File: README.md [hunk 4]
```diff
@@ -1 +1 @@
-# code-review
\ No newline at end of file
+# Youtube-review
```
Comment (Eckybrazzz) [inline README.md:1]: Please use a correct README.md file with all the information of you project. You can use this template to create a great README.md

https://github.com/microverseinc/readme-template

**Review by Eckybrazzz — COMMENTED** [review]
## Hello @Eckybrazzz ,

## Highlights :fire: 
- Your UI is looking amazing!
- Your code is highly organized and readable.

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 4 above)
Comment (rotimiazeez) [reply README.md:1]: I'd suggest you use emojis to make your comments friendlier 😎😜

File: index.html (hunk 1 above)
Comment (rotimiazeez) [reply index.html:36]: I'd suggest you always add a why to your comment for a better understanding of the problem😜😊

**Review by rotimiazeez — APPROVED** [review]
# Hi @Eckybrazzz 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**